guidelines/_sources/05_use-human-validation-for-llm-outputs.tex

```diff
@@ -6,57 +6,79 @@ \subsection{Use Human Validation for LLM Outputs}
+Where applicable, researchers \should conduct a power analysis to estimate the required sample size and ensure sufficient statistical power in their experiment design.
 
-For instance, ``Model-model agreement is high, for all criteria, especially for the three large models (GPT-4, Gemini, and Claude). Table I indicates that the mean Krippendorff’s $\alpha$ is 0.68-0.76. 
-Second, we see that human-model and human-human agreements are in similar ranges, 0.24-0.40 and 0.21-0.48
-for the first three categories.''~\cite{DBLP:journals/corr/abs-2408-05534}.
+In summary, when considering whether to employ human validation, researchers
+\begin{itemize}
+    \item \should employ human validation of LLM outputs whenever there is no established reference dataset or when automated metrics alone cannot fully capture the target qualities.
+    \item \should use systematic approaches to decide how human annotations can be replaced with LLM-generated ones.
+\end{itemize}
+
+When designing their methodology, researchers
+\begin{itemize}
+    \item \should plan human validation from the outset and integrate it into their study design.
+    \item \should use established reference models for comparing humans with LLMs.
+    \item \should use established measurement methods and instruments that are empirically validated.
+    \item \may aggregate validations from multiple human judges.
+    \item \should validate the agreement between multiple validators through inter-rater reliability measures such as Cohen's Kappa or Krippendorff's Alpha.
+    \item \should control for human confounding factors (e.g., experience/expertise in the relevant domain). 
+    \item \should conduct a power analysis to estimate the required sample size and ensure sufficient statistical power in their experiment design.
+\end{itemize}
+
+When reporting the results of the human validation, researchers
+\begin{itemize}
+    \item \must clearly define the construct that they are measuring in their \paper.
+    \item \must specify the methods used for human validation in their \paper.
+    \item \should include any measurement instruments (e.g., questionnaires, study tasks) in their \supplementarymaterial.
+    \item \should clearly describe their method of aggregation and document their reasoning for doing so in their \paper when aggregating measures from multiple subjects.
+\end{itemize}
 
 \subsubsection{Example(s)}
 
-As an example, Khojah et al.~\cite{DBLP:journals/pacmse/KhojahM0N24} augmented the results of their study using human measurement.
-Specifically, they asked participants to provide ratings regarding their experience, trust, perceived effectiveness and efficiency, and scenarios and lessons learned in their experience with ChatGPT.
+For instance, ``A subset of 20\% of the LLM-generated annotations was reviewed and validated by experienced software engineers to ensure accuracy. Using Cohen's Kappa, an inter-rater reliability of $\kappa = 0.90$ was reached.''
```

Comment:
would we want the number of software engineers to be included? e/g/ "validated by 4 ..."